go/test/endtoend/recovery/pitr/shardedpitr_test.go

```diff
@@ -525,7 +524,6 @@ func launchRecoveryTablet(t *testing.T, tablet *cluster.Vttablet, binlogServer *
 	tablet.MysqlctlProcess = *mysqlctlProcess
 	extraArgs := []string{"--db-credentials-file", dbCredentialFile}
 	tablet.MysqlctlProcess.InitDBFile = initDBFileWithPassword
-	tablet.VttabletProcess.DbPassword = mysqlPassword
```

Comment:
@shlomi-noach @mattlord @rohit-nayak-ps This line was crashing this test, so we never actually ran it and it was always passing because of the now removed `defer cluster.PanicHandler(nil)` call. Looks like the test might now be failing, but it means it has likely been failing forever / for a really long time. 

Any ideas on how to fix this?

Reply:
It's failing here:
```
func testTabletRecovery(t *testing.T, binlogServer *binLogServer, lookupTimeout, restoreKeyspaceName, shardName, expectedRows string) {
	recoveryTablet := clusterInstance.NewVttabletInstance("replica", 0, cell)
	launchRecoveryTablet(t, recoveryTablet, binlogServer, lookupTimeout, restoreKeyspaceName, shardName)

	sqlRes, err := recoveryTablet.VttabletProcess.QueryTablet(getCountID, keyspaceName, true)
	require.NoError(t, err)
	assert.Equal(t, expectedRows, sqlRes.Rows[0][0].String())
```

So my first guess would be that we need to wait for replication to catch up and for the expected result. Currently all it's doing is waiting for the tablet to become SERVING and then it queries it. Perhaps the tablet needs a second to replicate things. I would first just try adding a sleep of 30 seconds or something.

Reply:
This tests the old-and-unsupported-and-actually-legacy Google Ripple binlog server based PITR.

- RFC: https://github.com/vitessio/vitess/issues/16673
- https://vitess.io/docs/22.0/reference/features/recovery/#point-in-time-recovery-legacy-functionality-based-on-binlog-server

We should stop running this test altogether. Now is as good time as ever.

Reply:
@dbussink I've pushed a change to remove the test from running in shard `10`. For now this should fix this PR. Later on (on a different PR) I will purge the entire codebase and tests.